autrainer-configurations/dataset/MSPPodcast-EmoAct-wav.yaml

```diff
@@ -0,0 +1,29 @@
+# Important: should be used with inference_batch_size: 1
+id: MSPPodcast-EmoAct-wav
+_target_: autrainer.datasets.MSPPodcast
+
+path: data/MSPPodcast
+features_subdir: Audios
```

Comment:
This would need to be changed to `features_subdir: default` for all configurations that load the dataset in its default state (ref [Structure and Loading](https://autrainer.github.io/autrainer/modules/datasets.html)).

Reply:
Hmm, so how would you recommend doing it since the original download will create an "Audios" folder? I would make a symlink for me, but I'm not sure if that works for windows and would want to avoid a copying..

It's one of the things I like the least about our current state. This is only needed for preprocessing, right?

Reply:
Is it possible to move the data? If so i would say it makes sense to keep all datasets in the same structure and explain the transformation. Also ref https://github.com/autrainer/autrainer/pull/46#discussion_r1812740439

Reply:
Usually (at least for our infrastructure), it should be avoided, because not all users have write access. Also, more people might work with `MSP-Podcast` (or other data) without using `autrainer`. In that case, we don't want to destroy the original structure  for them

Reply:
In this case it is totally fine to symlink the dataset stucture by creating the `features_subdir` (as a symlink to the Audio dir) and also symlink the CSV (if the filepaths need adjustment its best to copy them).
However this would mean that preprocessed directories wont be located in the "globally shared" dataset location but in `${path}` next to the `${path}/default/` symlinked directory.

For all other users, the description of the download method (accessible via the docs) could outline the required manual preprocessing.
It is also possible that you provide a download method that transforms the dataset as needed (from the downloaded format to ours), and then just do the symlink yourself without using the download method. After the dataset root directory exists, the download method wont re-run and you will still be able to preprocess the dataset.